paddlecloud/paddlecloud/settings.py

```diff
@@ -275,18 +275,29 @@
 #       "mount_path" "/pfs/%s/home/%s/" # mount_path % ( dc, username )
 #    }
 #}
+FSTYPE_CEPHFS = "fstype"
+FSTYPE_HOSTPATH = "hostpath"
 DATACENTERS = {
```

Comment:
In the configuration file of the client, we should configure the `datacenters` and the `current-datacenter` which is configured by `settings.DATACENTERS`, so maybe `STORAGE_SERVICES` only means a storage configuration?